docs/_data/australia-2022-config.yaml

```diff
@@ -0,0 +1,182 @@
+    #   link: /sponsors/prospectus
+    # - text: Submit a Talk
+    #   link: /cfp
+    # - text: See the Schedule!
+    #   link: /schedule
+    # - text: Date and ticket information
+    #   link: /news/announcing-virtual-conf-dates-tickets
+    # - text: See the talks!
+    #   link: /speakers
+    # - text: Buy a Ticket!
+    #   link: /tickets
+ #    - text: Read our guide
+ #      link: /welcome-wagon
+ #    - text: See the schedule!
+ #      link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
+  corporate:
+    price: $200
+  independent:
+    price: $125
+  student:
+    price: $25
+
+sponsorship:
+  first_draft:
+    price: AUD $1,000
+  second_draft:
+    price: AUD $2,000
+  publisher:
+    price: AUD $4,000
+  patron:
+    price: AUD $7,000
+  keystone:
+    price: AUD $10,000
+
+date: # how do we handle these? human readable would be nice.
+  main: "**December 8-9, 2022**"
+  short: Dec 8-9, 2022
+  tickets_live: "July 2022"
+  month: May
+
+  # Conference is special for the index to show the combined conference days
+  conference:
+    event: Main Conference
+    date: Dec 8-9
+    summary: The main days of the conference. We will be running one main track, as well as <a href="/conf/australia/2022/lightning-talks/">Lightning Talks</a> & <a href="/conf/australia/2022/unconference/">Unconference</a>.
+    icon: conference
+
+  # These are listed in the schedule
+  day_one:
+    event: Conference Day 1
+    date: December 8
+    summary: This is the main event! Hear from lots of interesting folks about all things documentation. We will have talks all day, and unconference sessions running in parallel.
+    dotw: Thursday
+    talk_time: 11:00-18:00
+    unconference_time: 15:00-17:00
+    icon: conference
+  day_two:
+    event: Conference Day 2
+    date: December 9
+    summary: And the conference goes on! We are already starting to miss you. |:heart:|
+    dotw: Friday
+    talk_time: 11:00-18:00
+    unconference_time: 15:00-17:00
+    icon: conference
+  total_talk_days: 2
+
+about:
+  attendees: 150
+  summary:
+    "The main presentation track takes place during **December 2-3 (Thursday and Friday) from 12:00pm to 7:00pm AEDT**.
+    During the main event we also run an :doc:`/conf/australia/2021/unconference`."
+
+  social_venue: Spatial chat platform
+  venue:
+    "Online, https://hopin.to/. Your login will be emailed to you before the event"
+  photos:
+  mainroom: "Video area"
+  unconfroom: "Chat area"
+  projector_ratio: "16:9"
+  job_fair_room: "Sponsor area"
+
+cfp:
+  url: https://pretalx.com/wtd-australia-and-india-2021/cfp
+  ends: "August 31, 2021"           # 2 weeks to review and get confirmnations
```

Comment:
Although we are only doing a landing page now, I think it would good to set all the dates we know already, so that we don't accidentally leave old dates in there later.